Add definition for PFC in `Full combo` (#14600)
ppy/osu-wiki: wiki/Gameplay/Full_combo/en.md

**Full combo** (or *FC* for short) is a term used to describe a player obtaining the maximum possible [combo](/wiki/Beatmapping/Combo) on a [beatmap](/wiki/Beatmap). Full combos are scored by passing a beatmap with no misses<!-- TODO: link -->, [sliderbreaks](/wiki/Gameplay/Judgement/Slider_break), or dropped [slider ends](/wiki/Gameplay/Hit_object/Slider/Slidertail).

> Scores that lost combo only via dropped slider ends are widely considered by the community to be full combos. This differs from the game client and website's display. A play that do not have any combo breaks may also be referred to as a Perfect Full combo (or *PFC* for short, not to be confused with an [SS](/wiki/Gameplay/Grade) rank) to differentiate between those and scored with drops via slider ends.

Reviewer: ```suggestion
Scores that lost combo only via dropped slider ends are widely considered by the community to be full combos. This differs from the game client and website's display. A play that does not have any combo breaks may also be referred to as a Perfect Full combo (or *PFC* for short, not to be confused with an [SS](/wiki/Gameplay/Grade) rank) to differentiate between scores with and without dropped slider ends.
```

Author: shit mb lol thanks

> Scores that lost combo only via dropped slider ends are widely considered by the community to be full combos. This differs from the game client and website's display. A play that does not have any combo breaks may also be referred to as a Perfect Full combo (or *PFC* for short, not to be confused with an [SS](/wiki/Gameplay/Grade) rank) to differentiate between scores with and without dropped slider ends.

Reviewer 2: dropped slider ends don't break combo, so it wouldn't be a pfc anymore. you probably meant something like this

also full combo isn't capitalised

```suggestion
Scores that lost combo only via dropped slider ends are widely considered by the community to be full combos. This differs from the game client and website's display. A play with the maximum possible combo may also be referred to as a **perfect full combo** (or *PFC* for short, not to be confused with an [SS](/wiki/Gameplay/Grade) rank) to differentiate between scores with and without dropped slider ends.
```
